tests/unit/test_sde.py

```diff
@@ -213,14 +229,47 @@ def test_ssm_to_linear_drift(setup):
     A_p = -1 * ou_sde.decay * tf.ones_like(m)[..., None] * dt + tf.eye(1, dtype=DTYPE)
     b_p = tf.zeros_like(m) * dt
 
-    p_ssm = state_space_model_from_covariances(initial_mean=x0, initial_covariance=1e-1 * tf.ones_like(x0[..., None]),
-                                               state_transitions=A_p,
-                                               state_offsets=b_p,
-                                               process_covariances=tf.square(ou_sde.diffusion(m, t=None)) * dt)
+    p_ssm = state_space_model_from_covariances(
+        initial_mean=x0,
+        initial_covariance=1e-1 * tf.ones_like(x0[..., None]),
+        state_transitions=A_p,
+        state_offsets=b_p,
+        process_covariances=tf.square(ou_sde.diffusion(m, t=None)) * dt,
+    )
     linear_drift = LinearDrift()
     linear_drift.set_from_ssm(p_ssm, dt=dt)
 
     true_A = -1 * ou_sde.decay * tf.ones_like(m)[..., None]
     true_b = tf.zeros_like(m)
     np.testing.assert_allclose(true_A, linear_drift.A)
     np.testing.assert_allclose(true_b, linear_drift.b)
+
+
+def test_SSM_KL_with_grads_wrt_exp_params(with_tf_random_seed, num_transitions):
+    """
+    Test the SSM_KL_with_grads_wrt_exp_params function which calculates the KL divergence between two SSMs and the
+    gradients of it wrt the expectation params of the first SSM.
+    """
+    dist_p, _ = StateSpaceModelBuilder(
```

Comment:
docstrings to say this creates two different ssms
and that this construction is random